x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/support/TokensInvalidationResult.java

```diff
@@ -100,6 +93,5 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeStringCollection(invalidatedTokens);
         out.writeStringCollection(previouslyInvalidatedTokens);
         out.writeCollection(errors, StreamOutput::writeException);
-        out.writeVInt(attemptCount);
```

Comment:
we need to check the version of the stream here and if it is before the first version this change makes it into then we need to write an int

Reply:
Good catch. I'll be writing 5 which defaults to the max we allowed pre 7.1.0